python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -115,6 +115,37 @@ def test_flat_map_e2e(ray_start_cluster_enabled, enable_optimizer):
     assert ds.take_all() == [0, 0, 1, 1], ds
 
 
+def test_column_ops_e2e(ray_start_cluster_enabled, enable_optimizer):
```

Comment:
could we also add test to check `op.name`, `isinstance(physical_op, MapOperator)`, etc as above?